tests/conftest.py

```diff
@@ -591,18 +591,37 @@ def _generate_inputs_q2r():
 
 
 @pytest.fixture
-def generate_inputs_ph(fixture_sandbox, fixture_localhost, fixture_code, generate_remote_data, generate_kpoints_mesh):
+def generate_inputs_ph(
+    generate_calc_job_node, generate_structure, fixture_localhost, fixture_code, generate_kpoints_mesh
+):
     """Generate default inputs for a `PhCalculation."""
 
-    def _generate_inputs_ph():
+    def _generate_inputs_ph(with_output_structure=False):
```

Comment:
Could you add a quick explanation of this argument to the docstring? It wasn't immediately clear to me that this mean the `parent_folder` creator was the one who'd get an output structure. ^^ I was also thinking of maybe renaming the argument to make this more clear, but could only come up with ridiculously long options.